src/Libraries/Microsoft.AspNetCore.HeaderParsing/README.md

```diff
@@ -1 +1,130 @@
-README
+# Microsoft.AspNetCore.HeaderParsing
+
+This package provides services to for strongly typed header parsing and value caching.
+
+In particular:
+
+- There are many different headers with subtle format differences that are hard to parse correctly in all cases. This package provides predefined parsers for commonly used headers.
+- Your application may parse the same header value multiple times per request. This package automatically caches the parsed values of these headers to make your application faster.
+- This package also provides logging and metering of bad header values.
+
+## Getting started
+
+### Install the package
```

Comment:
Seems redundant?
```suggestion
## Install the package
```

Reply:
Followed @joperezr's README for consistency.

Reply:
Where?

Reply:
https://github.com/dotnet/extensions/pull/4512/files#diff-358a8dab66534c16517c7dc752a9f3926e6a3b4b3c2b713cd32d908d8be1f73bR5-R7